app/src/components/common/layout/header/index.tsx

```diff
@@ -164,6 +164,23 @@ const HeaderDropdown = styled(Dropdown)`
   height: 40px;
 `
 
+const MarketAndGovernanceNav = styled.div<{ disabled?: boolean }>`
+  background-color: ${props => (props.disabled ? props => props.theme.buttonSecondary.backgroundColor : 'transparent')};
+  ${props => (props.disabled ? 'pointer-events:none;' : '')};
+  border-radius: 32px;
+  color: ${props => (!props.disabled ? props.theme.colors.clickable : props.theme.colors.primary)};
+  cursor: pointer;
+  font-weight: 500;
+  letter-spacing: 1px;
+  line-height: 16.41px;
+  margin: 0px -60px 0px 68px;
```

Comment:
This is causing issue geronimo mentioned with margins....I think you should be able to do it without these margins....

Reply:
🙏